src/storm/solver/UncertaintyResolutionMode.h

```diff
@@ -0,0 +1,28 @@
+#ifndef STORM_UNCERTAINTYRESOLUTIONMODE_H
+#define STORM_UNCERTAINTYRESOLUTIONMODE_H
```

Comment:
```suggestion
#pragma once
```

Nowadays, we try to avoid preprocessor include guards (older files still have them)